stories/theme.DIS.introduction_disasters/carousel_content.json

```diff
@@ -11,5 +11,13 @@
         "src":"https://www.youtube.com/embed/TvPjAe4j6qQ",
         "title":"Ocean Flows",
         "caption":"This rainbow of colors show the water temperature on the oceans’ surface. The temperature variations correspond to the flow of currents on the surface. Blue is 32 degrees Fahrenheit, green is 50 -70 F, yellow is about 80 F, red is 90 F. This visualization came from computer models by a joint MIT/NASA-JPL project Estimating the Circulation and Climate of the Ocean, Phase II."
+    },{
+        "src":"https://www.youtube.com/embed/B7n62jOJJqI",
+        "title":"Atlantic Hurricane Wind Speed Plots",
+        "caption":"These simple visualizations are plots of time vs. wind speed for each tropical storm/hurricane of Atlantic Hurricane seasons from 1950 to the present. Horizontal lines indicate wind speed category thresholds. A line plot for each storm shows the storm's name and a marker at the peak wind speed.<p>\n\nMost named storms during the Atlantic hurricane season happen between June and November.  However, occasionally, storms develop outside of those ranges.\n\nFour versions of the plots are inluded:\n   1. May to December showig only the current year\n   2. May to December showing the current year and strong storms from previous years (ghosted out)\n   3. January to December showig only the current year\n   4. January to December showing the current year and strong storms from previous years (ghosted out)\n     \n<b>The plot for the current year automatically updates every 2 hours during hurricane season.</b>\nVisualizations by: Greg Shirah\nFor more information or to download this public domain video, go to https://svs.gsfc.nasa.gov/5072"
```

Comment:
html tags don't work
![image](https://github.com/NASA-IMPACT/veda-config-eic/assets/7830949/a5108ebf-a45e-4896-bd68-3d0437c6de58)

Reply:
Yep! Good catch, I haven't been able to find them all. I removed all the <a> and <p> but apparently not the <b> (this is coming from our svs.gsfc.nasa.gov's API for the captions on our page).

Reply:
removed!